packages/lib/src/account/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "dm3-lib-account",
+  "version": "1.0.0",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "module": "dist-backend/index.js",
+  "types": "dist/index.d.ts",
+  "exports": {
+      ".": {
+          "import": "./dist/index.js",
+          "require": "./dist/index.js"
+      },
+      "./dist.backend": {
+          "import": "./dist.backend/index.js",
+          "require": "./dist.backend/index.js"
+      }
+  },
+  "dependencies": {
+    "@types/libsodium-wrappers": "^0.7.10",
+    "ajv": "^8.11.0",
+    "axios": "^0.25.0",
+    "ethers": "^5.5.4",
+    "googleapis": "^100.0.0",
+    "libsodium-wrappers": "^0.7.10",
```

Comment:
Do we need this dependency outside of crypto?

Reply:
I've not reviewed the particular dependencies just copied the from the parent package.json